features/other_a/invite_request.feature

```diff
@@ -156,3 +156,24 @@ Feature: Invite requests
     When I follow "Delete"
     Then I should see "Invitation successfully destroyed"
       And "user1" should have "4" invitations
+
+  Scenario: Translated email is sent when invitation request is declined by admin
+    Given a locale with translated emails
+      And invitations are required
+      And the user "user1" exists and is activated
+      And the user "notuser1" exists and is activated
+      And the user "user1" enables translated emails
+      And all emails have been delivered
+    When I am logged in as "user1"
+      And as "user1" I request some invites
+      And I log out
+    When I am logged in as "notuser1"
+      And as "notuser1" I request some invites 
+      And I log out
+    When I view requests as an admin
+      And I press "Decline All"
+      And I log out
+    Then "user1" should be emailed
+      And the email to "user1" should be translated
```

Comment:
Could you add a step here that checks the subject of the email to make sure it's the right email? Same for the email to notuser1